Terraform/calibnet/main.tf

```diff
@@ -0,0 +1,30 @@
+terraform {
+  required_version = ">= 1.3"
+
+  backend "s3" {
+    bucket                      = "forest-test-state"
+    key                         = "terraform.tfstate"
+    region                      = "us-west-1"
+    endpoint                    = "nyc3.digitaloceanspaces.com"
+    skip_credentials_validation = true
+    skip_metadata_api_check     = true
+  }
+}
+
+module "calibnet" {
+  source = "../modules/forest_node"
+
+  digitalocean_token    = var.digitalocean_token
+  name                  = var.name
+  region                = var.region
+  backups               = var.backups
+  image                 = var.image
+  size                  = var.size
+  observability_name    = var.observability_name
+  protocol              = var.protocol
+  source_addresses      = var.source_addresses
+  attach_volume         = false
+  ssh_key               = var.ssh_key
```

Comment:
Let's use the SSH fingerprints from the entire ChainSafe organization. You can get them like this:
```
data "digitalocean_ssh_keys" "keys" {
  sort {
    key       = "name"
    direction = "asc"
  }
}
```

Reply:
Yes we could do this but this would mean that I redeploy the whole infrastructure again to add everyone's fingerprints - is that okay? Cos' I thought I had this conversation previously and we agreed it was okay to not have everyone have access to the droplet